Units/parser-cxx.r/bug1585745.cpp.d/expected.tags

```diff
@@ -3,4 +3,4 @@ Class5	input.cpp	/^class Class5 {$/;"	c	file:
 ~Class2	input.cpp	/^Class2::~ Class2() { }$/;"	f	class:Class2
 ~Class3	input.cpp	/^Class3:: ~Class3() { }$/;"	f	class:Class3
 ~Class4	input.cpp	/^Class4:: ~ Class4() { }$/;"	f	class:Class4
-~Class5	input.cpp	/^    public: ~ Class5() { }$/;"	f	class:Class5
+~Class5	input.cpp	/^    public: ~ Class5() { }$/;"	f	class:Class5	file:
```

Comment:
A class declared in a cpp file is never accessible from other files (well.. unless one includes one cpp file from another). So that's why I have added it. But again, can change.